src/ui/ContextSwitcher/ContextSwitcher.jsx

```diff
@@ -60,17 +54,10 @@ ModalSection.propTypes = {
   ModalControl: PropTypes.func,
 }
 
-function ContextItem({
-  context,
-  currentContext,
-  defaultOrgUsername,
-  setToggle,
-}) {
-  const { owner, pageName } = context
-  const orgUsername = owner?.username
-  const isActiveContext =
-    orgUsername?.toLowerCase() ===
-    currentContext?.owner?.username?.toLowerCase()
+function ContextItem({ context, defaultOrgUsername, setToggle }) {
+  const { owner } = useParams()
```

Comment:
Can you also pull out the `useParams` here, it's in the same vein of using `services` you're tying this component to the URL when it should be agnostic.

Reply:
on it! i was mainly focused on getting the use of active context right, but it looks like this component could use some more love